sycl/unittests/helpers/PiMockPlugin.hpp

```diff
@@ -945,6 +945,12 @@ mock_piextEventCreateWithNativeHandle(pi_native_handle nativeHandle,
   return PI_SUCCESS;
 }
 
+inline pi_result mock_piEnqueueTimestampRecordingExp(
+    pi_queue queue, pi_bool blocking, pi_uint32 num_events_in_wait_list,
```

Comment:
If you leave the parameters unnamed then unused warnings aren't a possibility. Not sure if that's the accepted style, but fewer warnings would be nice

Reply:
Though I agree and I don't think there's anything speaking against that style, we should try to stick with the style in the rest of the file and make a file-wide change if we want the style to change.

Reply:
Fair. Maybe that's a task I'll take on when I'm feeling energetic ;)